content/backend/graphql-elixir/3-mutations.md

```diff
@@ -11,6 +11,7 @@ Setting up mutations is as easy as queries, and you'll follow basically the same
 
 ```elixir(path=".../graphql-elixir/lib/community_web/schema.ex")
 mutation do
+  @desc “Create a new link”
```

Comment:
These are smart quotes, but they need to be dumb quotes.

Reply:
Good catch, thank you!